manage.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env python
+import os
+import sys
+
+if __name__ == "__main__":
+    os.environ.setdefault("DJANGO_SETTINGS_MODULE", "openedx_rocketchat.settings.production")
```

Comment:
Please, run from common, since we are not test settings. @andrey-canon

Reply:
ok